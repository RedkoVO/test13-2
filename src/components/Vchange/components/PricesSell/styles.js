export default () => ({
  root: {
    padding: '100px 0',
    width: '340px'
  },
  title: {
    paddingBottom: '20px',
    textAlign: 'center',
    fontSize: '33px',
    fontFamily: 'ubBold',

    '& span': {
      color: '#6283ff'
    }
  },
  wrTable: {
    width: '100%',
    borderCollapse: 'separate',
    borderSpacing: ' 0 7px',

    '& th': {
      textAlign: 'left'
    },
    '& tr': {
      height: '26px',
      fontSize: '14px',
      fontFamily: 'ubMedium'
    },
    '& th:first-child': {
      paddingLeft: '15px'
    }
  },
  wrItem: {
    '& td': {
      backgroundColor: '#f8f8f8'
    },

    '& td:first-child': {
      paddingLeft: '15px',
      borderTopLeftRadius: '10px',
      borderBottomLeftRadius: '10px'
    },
    '& td:last-child': {
      borderTopRightRadius: '10px',
      borderBottomRightRadius: '10px'
    }
  }
})
