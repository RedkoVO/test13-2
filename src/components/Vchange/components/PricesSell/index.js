import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import styles from './styles'

const PricesSell = ({ classes, pricesSellData }) => (
  <div className={classes.root}>
    <div className={classes.title}>
      <span>Sell</span> BTC: Prices
    </div>

    <table className={classes.wrTable} cellSpacing="0">
      <tbody>
        <tr>
          <th>Price (USDT)</th>
          <th>Ammount (BTC)</th>
          <th>Total (USDT)</th>
        </tr>
        {pricesSellData.map(item => (
          <tr className={classes.wrItem} key={item.id}>
            <td>{item.price}</td>
            <td>{item.ammount}</td>
            <td>{item.total}</td>
          </tr>
        ))}
      </tbody>
    </table>
  </div>
)

PricesSell.propTypes = {
  pricesSellData: PropTypes.array
}

export default withStyles(styles)(PricesSell)
