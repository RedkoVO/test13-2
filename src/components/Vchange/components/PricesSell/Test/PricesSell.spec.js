import * as React from 'react'
import renderer from 'react-test-renderer'

import PricesSell from '../'

const pricesSellData = [
  { id: 1, price: '3752.22', ammount: '1.0000', total: '3752.2200000' },
  { id: 2, price: '3752.22', ammount: '1.0000', total: '3752.2200000' },
  { id: 3, price: '3752.22', ammount: '1.0000', total: '3752.2200000' },
  { id: 4, price: '3752.22', ammount: '1.0000', total: '3752.2200000' }
]

it('<PricesSell /> renders correctly', () => {
  const tree = renderer
    .create(<PricesSell pricesSellData={pricesSellData} />)
    .toJSON()
  expect(tree).toMatchSnapshot()
})
