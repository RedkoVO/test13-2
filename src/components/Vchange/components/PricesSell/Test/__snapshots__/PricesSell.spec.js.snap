// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PricesSell /> renders correctly 1`] = `
<div
  className="PricesSell-root-1"
>
  <div
    className="PricesSell-title-2"
  >
    <span>
      Sell
    </span>
     BTC: Prices
  </div>
  <table
    cellSpacing="0"
    className="PricesSell-wrTable-3"
  >
    <tbody>
      <tr>
        <th>
          Price (USDT)
        </th>
        <th>
          Ammount (BTC)
        </th>
        <th>
          Total (USDT)
        </th>
      </tr>
      <tr
        className="PricesSell-wrItem-4"
      >
        <td>
          3752.22
        </td>
        <td>
          1.0000
        </td>
        <td>
          3752.2200000
        </td>
      </tr>
      <tr
        className="PricesSell-wrItem-4"
      >
        <td>
          3752.22
        </td>
        <td>
          1.0000
        </td>
        <td>
          3752.2200000
        </td>
      </tr>
      <tr
        className="PricesSell-wrItem-4"
      >
        <td>
          3752.22
        </td>
        <td>
          1.0000
        </td>
        <td>
          3752.2200000
        </td>
      </tr>
      <tr
        className="PricesSell-wrItem-4"
      >
        <td>
          3752.22
        </td>
        <td>
          1.0000
        </td>
        <td>
          3752.2200000
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
